### PR TITLE
Create category and seed

### PR DESCRIPTION
**Pull request by davidmetta**
category table with associations
seed file

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,343 @@
...
+10.times do
+  user = User.create(
+    username: Faker::FunnyName.name,
+    email: Faker::Internet.email,
+    password: '123456'
+    )
+end
+
+#categories
+cat = %q[politics sports history antiquity fiction economics celebrities]
```
Comment (ghost) [inline db/seeds.rb]: Lines 48-69 are the same as:
%w[politics ... celebrities].each { |cat| Category.create(name: cat) }

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,343 @@
...
+President Abbas, you proudly pay Palestinian terrorists who murder Jews. In fact, the more they slay, the more you pay. That’s in their law too. And you condemn Israel’s morality? You call Israel racist?
+This is not the way to peace.  This is not the way to achieve the peace we all want and need and to which Israel remain committed. This body should not be applauding the head of a regime that pays terrorists. The UN should condemn such a despicable policy.
+And the UN, the UN which brokered a ceasefire in 2014, should demand that Hamas release our fallen soldiers Oron Shaul and Hadar Goldin, who was kidnapped when Hamas violated that very ceasefire.
+Hamas should also release the two Israeli citizens it holds captive, Abera Mengistu and Hisham al Sayed.
+Ladies and gentlemen,
+Every time I stand here, I feel as I do today. I am privileged to stand here as the Prime Minister of the Jewish and democratic state of Israel.
+Some believe that Israel cannot be both Jewish and democratic. This is false. Israel is both, and
```
Comment (ghost) [inline db/seeds.rb]: speeches

Comment (ghost) [inline db/seeds.rb]: instead of line 309-313:
vote = Vote.create(value: 1)
vote.votable = contribution

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,343 @@
...
+#destroy everything
+User.destroy_all
+Speech.destroy_all
+Speaker.destroy_all
+Contribution.destroy_all
+Comment.destroy_all
+Vote.destroy_all
+Category.destroy_all
+
+puts '~~~destroying database~~~'
```
Comment (lepetitbelge) [inline db/seeds.rb]: would put this at line 9

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,343 @@
...
+
```
Comment (lepetitbelge) [inline db/seeds.rb:20]: would add something like: puts "creating users"

File: db/seeds.rb [hunk 5]
```diff
@@ -306,11 +286,8 @@
       )
     if contribution.persisted?
       1.times do
-        vote = Vote.create(
-          value: 1,
-          votable_type: 'contribution',
-          votable_id: contribution.id
-          )
+        vote = Vote.create(value :1)
```
Comment (ghost) [inline db/seeds.rb]: value: 1 (typo)

**Comment by ghost** [comment]
Ok for me!

**Pull request merged**